### PR TITLE
Another Round

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
# What does this PR do?
Starting through the gateway collections to tweak them and the readmes have applications ->role user assignments done, Updating the vars in tasks, defaults, meta, and adding in the loop controls.

### CODE REVIEW

File: roles/gateway_organizations/README.md [hunk 1]
```diff
@@ -43,7 +71,7 @@ Options for the `gateway_organizations` variable:
 
 ```json
 {
-  "organizations_list": [
+  "gateway_organizations": [
```
Comment (djdanielsson) [inline roles/gateway_organizations/README.md]: I do wonder if we should just call this organizations and same with the controller role

File: roles/gateway_applications/README.md [hunk 2]
```diff
@@ -46,7 +90,7 @@ Options for the `applications_list` variable:
 
 ```json
  {
-  "applications_list": [
+  "gateway_applications": [
```
Comment (djdanielsson) [inline roles/gateway_applications/README.md]: Similar to organizations I wonder if we should just call this applications

File: roles/gateway_organizations/README.md (hunk 1 above)
Comment (branic) [reply roles/gateway_organizations/README.md]: I agree with David. We shouldn’t use the term “gateway” at all.

File: roles/gateway_applications/README.md (hunk 2 above)
Comment (branic) [reply roles/gateway_applications/README.md]: I agree with David. We shouldn’t use the term “gateway” at all.

**Review by djdanielsson — APPROVED** [review]
Looking good

**Pull request merged**